externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -101,6 +113,10 @@ case class SparkSQLEngine(spark: SparkSession) extends Serverable("SparkSQLEngin
         exec,
         Duration(60, TimeUnit.SECONDS))
     })
+    engineSavePath.foreach { p =>
```

Comment:
Since `engineSavePath` is a temporary directory, why not use `spark tmp dir` directly?

Reply:
`engineSavePath` can be HDFS as the storage path, so it is not the same as tmp dir.

Reply:
> `engineSavePath` can be HDFS as the storage path, so it is not the same as tmp dir.

got it